source/docs/zero-to-robot/step-4/creating-benchtop-test-program-cpp-java.rst

```diff
@@ -84,27 +115,73 @@ Defining the variables for our sample robot
 
 .. tabs::
 
-   .. group-tab:: Java
+   .. group-tab:: PWM
 
-      .. remoteliteralinclude:: https://raw.githubusercontent.com/wpilibsuite/allwpilib/v2022.4.1/wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/gettingstarted/Robot.java
-         :language: java
-         :lines: 19-24
-         :linenos:
-         :lineno-start: 19
+      .. tabs::
 
-   .. group-tab:: C++
+         .. group-tab:: Java
 
-      .. remoteliteralinclude:: https://raw.githubusercontent.com/wpilibsuite/allwpilib/v2022.4.1/wpilibcExamples/src/main/cpp/examples/GettingStarted/cpp/Robot.cpp
-         :language: c++
-         :lines: 12-20
-         :linenos:
-         :lineno-start: 12
+            .. remoteliteralinclude:: https://raw.githubusercontent.com/wpilibsuite/allwpilib/v2022.4.1/wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/gettingstarted/Robot.java
+               :language: java
+               :lines: 19-36
+               :linenos:
+               :lineno-start: 19
 
-      .. remoteliteralinclude:: https://raw.githubusercontent.com/wpilibsuite/allwpilib/v2022.4.1/wpilibcExamples/src/main/cpp/examples/GettingStarted/cpp/Robot.cpp
-         :language: c++
-         :lines: 49-57
-         :linenos:
-         :lineno-start: 49
+         .. group-tab:: C++
+
+            .. remoteliteralinclude:: https://raw.githubusercontent.com/wpilibsuite/allwpilib/v2022.4.1/wpilibcExamples/src/main/cpp/examples/GettingStarted/cpp/Robot.cpp
+               :language: c++
+               :lines: 12-20
+               :linenos:
+               :lineno-start: 12
+
+            .. remoteliteralinclude:: https://raw.githubusercontent.com/wpilibsuite/allwpilib/v2022.4.1/wpilibcExamples/src/main/cpp/examples/GettingStarted/cpp/Robot.cpp
+               :language: c++
+               :lines: 49-57
+               :linenos:
+               :lineno-start: 49
+
+   .. group-tab:: CTRE
+
+      .. tabs::
+
+         .. group-tab:: Java
+
+            .. code-block:: java
+
+               public class Robot extends TimedRobot {
+                  private final WPI_TalonFX m_leftDrive = new WPI_TalonFX(0);
+                  private final WPI_TalonFX m_rightDrive = new WPI_TalonFX(1);
+                  private final DifferentialDrive m_robotDrive = new DifferentialDrive(m_leftDrive, m_rightDrive);
+                  private final Joystick m_stick = new Joystick(0);
+                  private final Timer m_timer = new Timer();
+
+         .. group-tab:: C++
+
+            Inside of ``Robot.h``
```

Comment:
```suggestion
         .. group-tab:: C++ (Header)
```
This might need other formatting/indenting changes, so take note if batch-merging this suggestion.